app/src/main/res/layout/component_autocomplete_textview.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
```

Comment:
I was thinking of merge because it inherits from RelativeLayout in the Kotlin code, no?